Machine.h

```diff
@@ -94,4 +96,8 @@ void Machine_scan(Machine* this);
 
 void Machine_scanTables(Machine* this);
 
+double Machine_getGpuUsage(const Machine* super, double values[]);
```

Comment:
Passing unbounded arrays is dangerous. Also, the `Machine*` can just be extended to hold all the information for the GPU meter as necessary.

```suggestion
double Machine_getGpuUsage(const Machine* super);
```